Event/RefreshAuthenticationFailureEvent.php

```diff
@@ -13,14 +13,15 @@
 
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Security\Core\Exception\AuthenticationException;
+use Symfony\Contracts\EventDispatcher\Event;
 
 class RefreshAuthenticationFailureEvent extends Event
 {
     private AuthenticationException $exception;
 
-    private ?Response $response;
+    private Response $response;
```

Comment:
SF6 typehints the response as non-nullable in authentication success and failure handlers, so the events shouldn't allow nulllability either